### PR TITLE
feat: Add support for Android 10 and Android X

### PR DESCRIPTION
**Pull request by GuillaumeSE**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/Mixins/Android/FrameworkElementMixins.tt [hunk 1]
```diff
@@ -8,7 +8,11 @@
 	AddClass("Windows.UI.Xaml.Controls", "Image", hasNewMaxWidthHeight: true, hasAutomationPeer: true);
 	AddClass("Uno.UI.Controls.Legacy", "HorizontalGridView");
 	AddClass("Uno.UI.Controls.Legacy", "HorizontalListView");
+#if __ANDROID_29__
```
[the remark this answers is not in the document]
Comment (carldebilly) [reply src/Uno.UI/Mixins/Android/FrameworkElementMixins.tt]: https://docs.microsoft.com/en-us/xamarin/cross-platform/app-fundamentals/building-cross-platform-applications/platform-divergence-abstraction-divergent-implementation#android

> _Each API level includes all the ‘lower’ level symbols._

**Comment by jeromelaban** [comment]
@GuillaumeSE You'll likely need to update the second commit to conform to conventional commits

**Comment by jeromelaban** [comment]
It looks like at this point, it's not possible to UI Test using Android 10 emulators, we'll add that once this is fixed in Xamarin.UITest.

**Pull request merged**